frontend/src/scenes/Game.ts

```diff
@@ -385,7 +360,7 @@ export default class Game extends Phaser.Scene {
     const sessionId = serverBomb.owner.sessionId;
 
     // 自分のボムは表示しない
-    if (this.currentPlayer.isEqualSessionId(sessionId)) return;
+    if (this.currentPlayer.isEqualSessionId(sessionId) === true) return;
```

Comment:
[@typescript-eslint/strict-boolean-expressions](https://typescript-eslint.io/rules/strict-boolean-expressions) の解消法がいまいち分からなかったので

Reply:
と思ったらローカルとactionsでのlintが言ってることが違った
github actions Lint/Format / frontend 
> Error:   350:9  error  This expression unnecessarily compares a boolean value to a boolean instead of using it directly  @typescript-eslint/no-unnecessary-boolean-literal-compare